test/Microsoft.ML.Tests/TextLoaderTests.cs

```diff
@@ -270,15 +270,15 @@ public class Input
         public class InputWithUnderscore
         {
             [Column("0")]
-            public string String_1;
+            public ReadOnlyMemory<char> String_1;
 
             [Column("1")]
-            public float Number_1;
+            public ReadOnlyMemory<char> Number_1;
```

Comment:
This seems wrong - it was `float` before, and the property is called `Number_1`.